Source/IO.cpp

```diff
@@ -279,6 +283,15 @@ PeleC::checkPoint(
 {
   amrex::AmrLevel::checkPoint(dir, os, how, dump_old);
 
+#ifdef PELEC_USE_SPRAY
+  if (theSprayPC() != nullptr) {
+    bool is_checkpoint = true;
+    int write_ascii = 0; // Not for checkpoints
+    theSprayPC()->SprayParticleIO(
+      level, is_checkpoint, write_ascii, dir, PeleC::spray_fuel_names);
```

Comment:
Can you remove the variables and instead of `level, is_checkpoint, write_ascii, dir, PeleC::spray_fuel_names);` just do `level, true, 0, dir, PeleC::spray_fuel_names);`

Reply:
Yeah and the API might change in which case they become obsolete